integration-commons/src/test/java/org/symphonyoss/integration/exception/config/ForbiddenUserExceptionTest.java

```diff
@@ -0,0 +1,46 @@
+package org.symphonyoss.integration.exception.config;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Unit tests for {@link ForbiddenUserException}.
+ * Created by crepache on 23/06/17.
+ */
+public class ForbiddenUserExceptionTest {
+
+  @Test
+  public void testForbiddenUserException() {
+    ForbiddenUserException exception = new ForbiddenUserException();
+    String resultMessage = exception.getMessage();
+    String expectedMessage = "\n" +
```

Comment:
You could create a private method to build these String messages and avoid repeated code.